index.html

```diff
@@ -44,6 +44,217 @@ <h3>I'am a Software Developer</h3>
         </ul>
       </section>
     </section>
+      
+    <section class="container1">
+      <section class="card1">
+        <h2>My Recent Works</h2>
+
+        <hr>
+        
+        <img src="Placeholder.png" alt="">
+
+        <div class="multiple">
+          <h3>Multi-Post Stories</h3>
+        
+          <p>
+            A daily selection of privately personalized 
+            reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy 
+            text ever since the 1500s, when an 
+            unknown printer took a standard dummy 
+            text.
+          </p>
+
+          <ul class="buttons">
+            <li><button class="button">css</button></li>
+            <li><button class="button">html</button></li>
+            <li><button class="button">bootstrap</button></li>
+            <li><button class="button">Ruby</button></li>
+          </ul>
+          <br>
+        
+          <button class="orange-button">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>

```

Comment:
- [ ] Kindly follow the typography of the page according to the template design. Adjust the font used here to that of the design.

- [ ] The list of projects should use a `ul` tag to group them, and `li` per project.
`ul`-`li` orders the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group, this helps with proper accessibility and webpage optimisation.
please try to use ul and li for semantic correctness whenever you are trying to list items. 

- [ ] Also increase the spacing between each project card.


- [ ] Good job using `ul` -`li` to list the technologies.